test/services/nexboard/MockServer.js

```diff
@@ -0,0 +1,123 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const logger = require('../../../src/logger');
+
+// TODO: use config
+const {
+	NEXBOARD_URL = 'https://nexboard.nexenio.com/',
+	NEXBOARD_URI = 'portal/api/v1/public/',
+} = process.env;
+
+module.exports = function MockServer({
+	nexUrl = NEXBOARD_URL,
+	uri = NEXBOARD_URI,
```

Comment:
I think you need to explain this to me in person... why does the mock server need the actual URIs?